### DISCUSSION TITLE
Blurred text

### DISCUSSION THREAD

**Discussion by Xant1k** [uBlockOrigin/uBlock-issues#2592] [Q&A] [answered]
https://i.postimg.cc/d3szj7dL/vi-Wz-Iu-Pu-Dr.png

How to fix it? What javascript should I block?

rep on which this happens
https://github.com/freepn/fpnd
https://github.com/ChristianGalla/PidginWinToastNotifications

**Comment by ghost** [comment]
![image](https://user-images.githubusercontent.com/886325/231109445-fa208d78-55d2-4944-b857-fd3a9e4e5d6d.png)

It's just ClearType vs grayscale antialiasing. Browser issue?

**Comment by krystian3w** [comment]
> What javascript should I block?

Block all or one by one how you think it sits in Microsoft's JavaScript.

**Comment by mtxadmin** [comment]
I didn't see blurred text too.

Thought you meant some filter=blur case

**Reply by Xant1k** [reply]
look this https://imgbox.com/TqjtrNsu
this happens on many sites including reddit, github

**Reply by gorhill** [reply, marked as answer]
Surely that is not a uBO issue.
